### DISCUSSION TITLE
image rendering asynchrony: issue with preload?

### DISCUSSION THREAD

**Discussion by cainesap** [jspsych/jsPsych#2231] [Q&A]
Hello,
I'm using jsPsych to show 2 images side-by-side, and we've noticed that sometimes there's a delay with rendering one of the images, which is obviously a bit of a confound for the reaction times we're collecting.. There was a recent [conversation on the google group](https://groups.google.com/g/jspsych/c/q1hOBVHuynY) which made me look at the network tab in the developer console. The images are preloaded, or should be, but it still seems that the images are being requested from the server for each trial. I can post more information, such as code snippets, screenshots, or a reproducible example. But I wondered first of all if anyone knows how to investigate what seems to be a preload issue, or has experienced something similar?
best wishes and thank you in general for jsPsych! Andrew

**Comment by jodeleeuw** [comment]
Hi @cainesap,

Have you seen the conversation in #1548, particularly near the bottom of the long thread? Does that apply to your situation?

**Comment by cainesap** [comment]
Thank you very much @jodeleeuw : I think it could do! I will try [the suggested diagnosis](https://github.com/jspsych/jsPsych/issues/1548#issuecomment-915611152) and check on server settings (if I understand correctly the outcome of the thread) . I'll update here as soon as I can
